src/main/java/com/yscope/logging/log4j1/AbstractBufferedRollingFileAppender.java

```diff
@@ -490,9 +503,11 @@ public void addShutdownRequest () {
      * @param logRolloverTimestamp The approximate timestamp of when the target
      * log file was rolled over
      * @param deleteFile Whether the log file can be deleted after syncing.
+     * @param metadata The metadata map object to be included in the sync request.
```

Comment:
`Metadata to synchronize with the file`?

Reply:
Modified to `Extra metadata for the file`